app/controllers/suggest_a_register_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SuggestARegisterController < ApplicationController
+  include FormHelpers
+  helper_method :government_orgs_local_authorities
+  invisible_captcha only: :create_feedback, honeypot: :spam
+
+  def index
+    @government_organisations = government_orgs_local_authorities
+  end
+
+  def check
+    if params['suggest_a_register']['gov_what_part_of_government'].present?
+      redirect_to suggest_a_register_complete_path
+    else
```

Comment:
Another potential problem with having a separate step to collect the organisation is there seems to be no way to link up the data about organisations with the actual form responses?

Reply:
The spec asked for this question to be mandatory - the reasoning is that we can only help and host government registers. (The definition of government being the interesting thing.)

A yes / no answer would serve the same purpose, allow people to use their interpretation of government when answering, and avoid duplication with the form later. I think that's a conversation worth having.

We could parse email addresses, but I'd rather not since we then run into the duplication in the main form again. It's also brittle, and we then run the risk of excluding organisations with non-`.gov.uk` email addresses, such as the NHS.